pyproject.toml

```diff
@@ -77,7 +77,7 @@ langchain-anthropic = "^0.1.6"
 langchain-astradb = "^0.3.0"
 langchain-openai = "^0.1.1"
 zep-python = { version = "^2.0.0rc5", allow-prereleases = true }
-langchain-google-vertexai = "^1.0.3"
+langchain-google-vertexai = { extras = ["langchain-google-vertexai"], version = "^1.0.3", optional = true }
```

Comment:
this seems wrong, langchain-google-vertexai doesn't have an extra called "langchain-google-vertexai"

Reply:
I wasn't as careful as I should have been in implementing these changes. As a result, I have decided to revert the modifications